apis/apps/v1alpha1/advancedcronjob_types.go

```diff
@@ -66,11 +71,11 @@ type AdvancedCronJobSpec struct {
 type CronJobTemplate struct {
 	// Specifies the job that will be created when executing a CronJob.
 	// +optional
-	JobTemplate *batchv1beta1.JobTemplateSpec `json:"jobTemplate" protobuf:"bytes,1,opt,name=jobTemplate"`
+	JobTemplate *batchv1beta1.JobTemplateSpec `json:"jobTemplate,omitempty" protobuf:"bytes,1,opt,name=jobTemplate"`
```

Comment:
Added `omitempty` for `JobTemplate` and `BroadcastJobTemplate` as it is an optional field.